index.js

```diff
@@ -93,9 +93,9 @@ module.exports = function (log, indexesPath) {
     }
   })
 
-  const B_TIMESTAMP = Buffer.from('timestamp')
-  const B_SEQUENCE = Buffer.from('sequence')
-  const B_VALUE = Buffer.from('value')
+  const B_TIMESTAMP = bipf.allocAndEncode('timestamp')
```

Comment:
We use the convention `B_FOO` for `Buffer.from('foo')` so perhaps we can name these `FOO_BIPF` instead

Reply:
I went with BIPF_